network/default.go

```diff
@@ -1058,28 +1058,37 @@ func (n *network) processNetChan(listener tunnel.Listener) {
 					// lookup best routes for the services in the just received route
 					q := []router.QueryOption{
 						router.QueryService(route.Service),
-						router.QueryStrategy(router.AdvertiseBest),
+						router.QueryStrategy(n.router.Options().Advertise),
 					}
-					bestRoutes, err := n.options.Router.Table().Query(q...)
+
+					routes, err := n.options.Router.Table().Query(q...)
```

Comment:
using n.router and n.options.Router in another place, should be consistent

Reply:
We should actually be more consistent with this in general not just in this piece of code 😅